packages/components/pie-modal/test/helpers/page-object/selectors.ts

```diff
@@ -20,6 +20,10 @@ const modal = {
             description: 'The selector for the modal content',
             dataTestId: 'modal-content-inner',
         },
+        description: {
+            description: 'The selector for the modal description',
```

Comment:
I wonder if this could just be a JSON file a bit like this:

Not for this PR but just something to consider in future to see if it might simplify the imports, etc.

```json
{
    "container": {
        "description": "The selector for the modal container",
        "dataTestId": "pie-modal"
    },
    "backButton": {
        "description": "The selector for the back button",
        "dataTestId": "modal-back-button"
    },
    "closeButton": {
        "description": "The selector for the close button",
        "dataTestId": "modal-close-button"
    },
    "header": {
        "description": "The selector for the modal header",
        "dataTestId": "modal-header"
    },
    "content": {
        "description": "The selector for the modal content",
        "dataTestId": "modal-content-inner"
    },
    "description": {
        "description": "The selector for the modal description",
        "dataTestId": "modal-description"
    }
}
```

Reply:
Then you might be able to do something like `import selectors from '/path/to/modal/selectors.json';`

Although then that only cuts it down from `modal.selectors.content.description` to `selectors.content.description`, so maybe the easier thing to do would just be to rename the export and then remove one level of nesting.

```ts
const selectors = {
    container: {
        description: 'The selector for the modal container',
        dataTestId: 'pie-modal',
    },
    backButton: {
        description: 'The selector for the back button',
        dataTestId: 'modal-back-button',
    },
    closeButton: {
        description: 'The selector for the close button',
        dataTestId: 'modal-close-button',
    },
    header: {
        description: 'The selector for the modal header',
        dataTestId: 'modal-header',
    },
    content: {
        description: 'The selector for the modal content',
        dataTestId: 'modal-content-inner',
    },
    description: {
        description: 'The selector for the modal description',
        dataTestId: 'modal-description',
    },
};

export {
    selectors,
};
```